ui/apps/moderation/src/extensions/moderate-modal.tsx

```diff
@@ -68,7 +68,7 @@ const ModerateModalComponent = (props: RootExtensionProps) => {
       footerUrl1={`${window.location.protocol}//${window.location.host}/@akashaorg/app-legal/code-of-conduct`}
       cancelLabel={t('Cancel')}
       errorText={moderateMutation.error ? `${moderateMutation.error}` : ''}
-      user={loginQuery.data?.pubKey || ''}
+      user={loginQuery.data?.id || ''}
```

Comment:
also here, loginQuery should be replaced with loggedProfileData coming from the new generated hook, look in app-akasha-integration/app-routes for an example